src/test/java/unit/Syntax.java

```diff
@@ -0,0 +1,41 @@
+package unit;
+
+import static org.hamcrest.CoreMatchers.containsString;
+import static org.hamcrest.MatcherAssert.assertThat;
+
+import harness.TestCommand;
+import harness.TestCommand.Result;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Tag;
+import org.junit.jupiter.api.Test;
+
+/** Tests for syntax variations */
+@Tag("unit")
+public class Syntax {
```

Comment:
This should extend one of the classes in `harness.baseclasses`, probably `ClearContextUnit`.

Reply:
done